CMakeLists.txt

```
@@ -25,6 +25,15 @@ include(cmake/go.cmake)
 
 enable_language(C)
 
+# Configure entropy source in case of FIPS build
+if(FIPS)
+  if(USE_FIPS_ENTROPY_SOURCE_PASSIVE)
```

Would it be better to inverse this if? 
```suggestion
  if(USE_JITTER)
```

Then you could also enforce you can't set USE_JITTER if FIPS isn't set as well. Otherwise we need to decide what does USE_FIPS_ENTROPY_SOURCE_PASSIVE if FIPS isn't set.

---

Not sure I understand your point. I'm gonna flip this later on to passive being the default - as you suggest.

Is it easier to just make this flip now?

---

My concern is I don't want to switch the behavior because that's another "breaking" change. But since this is the FIPS release and customers have to follow the specific build instructions and use a specific version of code it's probably fine.